src/components/Icon/Arrow.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import { SVGProps } from "react";
+
+export interface ArrowIconProps extends SVGProps<SVGSVGElement> {
+  variant?: "down" | "up" | "left" | "right"
```

Comment:
We should either have this required, or have a default variant (potentially left?)

Reply:
Otherwise "down" is not needed as a variant, as that's the default anyway right?

Reply:
Done!

Reply:
Thanks, added a new comment, sorry! :)

https://github.com/Major-Eth/yearn-web-lib/pull/21#discussion_r819236593